package/spa-config.json

```diff
@@ -1,28 +1,31 @@
 {
   "microfrontends": {
-    "@openmrs/esm-login-app": "next",
-    "@openmrs/esm-primary-navigation-app": "next",
-    "@openmrs/esm-implementer-tools-app": "next",
     "@openmrs/esm-devtools-app": "next",
-    "@openmrs/esm-patient-chart-app": "next",
-    "@openmrs/esm-home-app": "next",
-    "@openmrs/esm-patient-search-app": "next",
     "@openmrs/esm-drugorder-app": "next",
+    "@openmrs/esm-home-app": "next",
+    "@openmrs/esm-implementer-tools-app": "next",
+    "@openmrs/esm-login-app": "next",
     "@openmrs/esm-patient-allergies-app":"next",
-    "@openmrs/esm-patient-vitals-app": "next",
-    "@openmrs/esm-patient-notes-app": "next",
-    "@openmrs/esm-patient-list-app": "next",
+    "@openmrs/esm-patient-appointments-app": "next",
+    "@openmrs/esm-patient-attachments-app": "next",
     "@openmrs/esm-patient-banner-app": "next",
-    "@openmrs/esm-patient-conditions-app": "next",
-    "@openmrs/esm-patient-programs-app": "next",
     "@openmrs/esm-patient-biometrics-app": "next",
-    "@openmrs/esm-patient-appointments-app": "next",
-    "@openmrs/esm-patient-immunizations-app": "next",
-    "@openmrs/esm-patient-forms-app": "next",
+    "@openmrs/esm-patient-chart-app": "next",
     "@openmrs/esm-patient-clinical-view-app": "next",
-    "@openmrs/esm-patient-registration-app": "next",
-    "@openmrs/esm-patient-attachments-app": "next",
+    "@openmrs/esm-patient-conditions-app": "next",
+    "@openmrs/esm-patient-immunizations-app": "next",
+    "@openmrs/esm-patient-list-app": "next",
     "@openmrs/esm-patient-medications-app": "next",
-    "@openmrs/esm-patient-test-results-app": "next"
+    "@openmrs/esm-patient-notes-app": "next",
+    "@openmrs/esm-patient-programs-app": "next",
+    "@openmrs/esm-patient-registration-app": "next",
+    "@openmrs/esm-patient-search-app": "next",
+    "@openmrs/esm-patient-test-results-app": "next",
+    "@openmrs/esm-patient-vitals-app": "next",
+    "@openmrs/esm-primary-navigation-app": "next"
+  },
+  "not-working-mfs": {
```

Comment:
Is this doing something specific? or just a place to stack ESMs we don't use.

Reply:
It's just a place to store them. It's intended to be temporary and mostly as a reminder to myself to bug the MF guys about them.